test-result-summary-client/src/TrafficLight/TrafficLight.jsx

```diff
@@ -50,6 +54,42 @@ function TrafficLight() {
             );
         }
     }
+
+    const buildJavaVersionMap = async (data, buildType, originBuildType) => {
+        let javaVersionMap = new Map();
+        for (const childData of data) {
+            childData.buildType = buildType;
+            const buildNameWithPlatform = childData.buildName.slice(
+                0,
+                lastIndexOf('_')
```

Comment:
I am not sure it is correct to use Lodash _.lastIndexOf() without an array.

https://www.geeksforgeeks.org/javascript/lodash-_-lastindexof-method/

It should be `buildNameWithPlatform.lastIndexOf('_')`